packages/@sanity/client/README.md

```diff
@@ -23,14 +23,25 @@ const sanityClient = require('@sanity/client')
 const client = sanityClient({
   projectId: 'your-project-id',
   dataset: 'bikeshop',
-  token: 'sanity-auth-token', // or leave blank to be anonymous user
+  apiVersion: '2019-01-29', // use current UTC date - see "specifying API version"!
+  token: 'sanity-auth-token', // or leave blank for unauthenticated usage
   useCdn: true // `false` if you want to ensure fresh data
 })
 ```
 
 `const client = sanityClient(options)`
 
-Initializes a new Sanity Client. Required options are `projectId` and `dataset`.
+Initializes a new Sanity Client. Required options are `projectId`, `dataset` and `apiVersion`. Setting a value for `useCdn` is encouraged.
+
+### Specifying API version
+
+Sanity uses ISO dates (YYYY-MM-DD) in UTC timezone for versioning. The explanation for this can be found [in the documentation](http://sanity.io/help/api-versioning)
+
+In general, unless you know what API version you want to use, you'll want to set it to todays UTC date. By doing this, you'll get all the latest bugfixes and features, while preventing any timezone confusion and locking the API to prevent breaking changes.
+
+**Note**: Do not be tempted to use a dynamic value for the `apiVersion`. The whole reason for setting a static value is to prevent unexpected, breaking changes.
+
+In future versions, specifying an API version will be required. For now (to maintain backwards compatiblity) not specifying a version will trigger a deprecation warning and fall back to using `v1`.
```

Comment:
Is it required or not? First paragraph says `apiVersion` is required, here we say it's optional.

Reply:
"Almost required"? :)